dipy/reconst/ivim.py

```diff
@@ -226,13 +252,17 @@ def __init__(self, gtab, split_b_D=400.0, split_b_S0=200., bounds=None,
             raise ValueError(b0_s)
 
         ReconstModel.__init__(self, gtab)
-        self.split_b_D = split_b_D
-        self.split_b_S0 = split_b_S0
-        self.bounds = bounds
-        self.two_stage = two_stage
-        self.tol = tol
-        self.options = options
-        self.x_scale = x_scale
+        self.args = args
+        self.kwargs = kwargs
+        self.fit_method = fit_method
+        self.split_b_D = kwargs.get('split_b_D', 400.0)
+        self.split_b_S0 = kwargs.get('split_b_S0', 200.)
+        self.bounds = kwargs.get('bounds', None)
+        self.two_stage = kwargs.get('two_stage', True)
+        self.tol = kwargs.get('tol', 1e-15)
+        self.options = kwargs.get('options', {'gtol': 1e-15, 'ftol': 1e-15,
```

Comment:
What happens if I give this parameter: `options={'eps': 1e15}` ?

The user does not know what keyword should be in options. This is why I think it will be better to be explicit on the `__init__`and then, build your `options` variable.

Reply:
Done!